### DISCUSSION TITLE
[Feature] Partner sharing pictures to show up in photos

### DISCUSSION THREAD

**Discussion by kaczkalolo** [immich-app/immich#4926] [Feature Request]
### The feature

Could You please make all the shared pictures will be shown on main photos tab like it is in Google photos so I would not have to go to shared. I wish all these pictures were visible on the main tab with all my pictures and also in shared. 

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #2846
